datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py

```diff
@@ -288,6 +287,25 @@ def create_scraper_configuration(self, instance=None):
         if config['metadata_metric_name'] and config['metadata_label_map']:
             config['_default_metric_transformers'][config['metadata_metric_name']] = self.transform_metadata
 
+        # Set up the HTTP wrapper for this endpoint
+
+        # TODO: Deprecate this behavior
+        if config['ssl_verify'] is False:
+            config.setdefault('tls_ignore_warning', True)
+
+        http_handler = self.http_handlers[endpoint] = RequestsWrapper(
+            config, self.init_config, self.HTTP_CONFIG_REMAPPER, self.log
+        )
```

Comment:
I think we need a test for the logic around the http handlers. What do you think?

Reply:
We can in subsequent PRs, I'm trying to keep the diff small